plone/app/contentrules/exportimport/rules.py

```diff
@@ -113,54 +100,16 @@ def export_field(self, doc, field):
 
         child = doc.createElement('property')
         child.setAttribute('name', field.__name__)
-
-        if value is not None:
-            if ICollection.providedBy(field):
-                for e in sorted(value):
-                    list_element = doc.createElement('element')
-                    list_element.appendChild(doc.createTextNode(str(e)))
-                    child.appendChild(list_element)
-            else:
-                child.appendChild(doc.createTextNode(six.text_type(value)))
-
+        
+        node = valueToElement(field, value)
+        if node.text:
+            child.appendChild(doc.createTextNode(six.text_type(node.text)))
+        # Assumes there are not other text nodes and we can throw away the parent node    
+        for node in node.iterchildren():
+            xml = etree.tostring(node, encoding="utf8")
+            child.appendChild(minidom.parseString(xml).firstChild)
```

Comment:
I am confused? Why ``minidom`` here if we have `lxml` already? If it is needed: at least a comment would help here

Reply:
Because export import code is using minidom already and rewriting all of it is not worth it.